library/core/src/ptr/mut_ptr.rs

```diff
@@ -78,7 +78,7 @@ impl<T: ?Sized> *mut T {
     /// Creates a pointer from its raw bits.
     ///
     /// This is equivalent to `as *mut T`, but is more specific to enhance readability.
-    /// The inverse method is [`Self::to_bits`].
+    /// The inverse method is [`to_bits`](#method.to_bits-1).
```

Comment:
Note that the number disambiguates between `*const` and `*mut`, though I don't know if the number is stable. If you don't want to disambiguate, you can also use `[`to_bits`](pointer::to_bits)` as an intra-doc link.

Reply:
I also don't know if it's stable.  I put it this way as it work for the current build.  Based on [the zulip conversation](https://zulip-archive.rust-lang.org/stream/266220-rustdoc/topic/Intra-doc.20links.20for.20pointers.3F.html#262644521) there's no great solution right now, so I think I'll just leave it like this.  Hopefully one day it can go back to just ```[`Self::to_bits`]```.